packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md

```diff
@@ -36,6 +36,14 @@ This page provides the API rate limits for authentication and end-user activitie
 | `/api/{apiVersion}/radius`                                                                                             | 100              | 600              | 600     | 600        | 600                   |
 | `/idp/idx`                                                                                                             | 100              | 600              | 600     | 600        | 500                   |
 | `/login/token/redirect`                                                                                                             | 100              | 600              | 600     | 600        | 600                   |
+| <ApiLifecycle access="ie" /> **Identity Engine APIs:**                                                                            |               |               |      |         |                    |
+| `/idp/idx`                                                                                                     | 100              | 600              | 600     | 600        | 1,200                   |
```

Comment:
shouldn't this last column also be a value of 600?

Reply:
Fixed.